package.json

```diff
@@ -10,11 +10,13 @@
   },
   "scripts": {
     "test": "eslint --config=.eslintrc.js --cache --cache-file=eslint-cache/base js/*.js data/graphics.js && eslint --config=githooks/.eslintrc.js --cache --cache-file=eslint-cache/build githooks/update githooks/build-indexes",
-    "fix": "eslint --config=.eslintrc.js --fix js/*.js data/graphics.js && eslint --config=githooks/.eslintrc.js --fix githooks/update githooks/build-indexes"
+    "fix": "eslint --config=.eslintrc.js --fix js/*.js data/graphics.js && eslint --config=githooks/.eslintrc.js --fix githooks/update githooks/build-indexes",
+    "testcafe": "githooks/build-indexes && testcafe chrome test/"
```

Comment:
Because it clones Pokemon Showdown repository and opens a web browser without closing it.